packages/generator-open-wc/generators/building-webpack/templates/_package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
```

Comment:
I think for your generators we will need to have a concept for all the scripts we will offer in the end
```
start => should be owc-dev-server? (in the end - not part of this generator)
start:build, start:building? => webpack-dev-server?
start:dist, start:built, start:constructed? => serving the dist (build vs built?)
```

I think I would go with `start:build`, `start:dist`  in this generator (+ rest stays as is)

Reply:
what about `"start:dev": "webpack-dev-server --mode development --open"` and `"start:prod": "http-server dist/ -o"`?

Reply:
uh `start:prod` is nice 👍 

`start:dev` would suggest that just `start` is not for development...
`start:build` is sort of start the building server

but I have no super strong opinion about it...

Reply:
Lets go with this for now?
```
    "start:dev": "webpack-dev-server --mode development --open",
    "start:dev:es5": "webpack-dev-server --mode development --es5",
    "start:prod": "http-server dist/ -o",
```